README.Rmd

```diff
@@ -42,73 +42,67 @@ knitr::opts_chunk$set(
 [![rstudio mirror downloads](https://cranlogs.r-pkg.org/badges/rredlist)](https://github.com/metacran/cranlogs.app)
 [![cran version](https://www.r-pkg.org/badges/version/rredlist)](https://cran.r-project.org/package=rredlist)
 
-`rredlist` is an R client for the IUCN Red List (http://apiv3.iucnredlist.org/api/v3/docs). 
+`rredlist` is an R client for the IUCN Red List (<https://apiv3.iucnredlist.org/api/v3/docs>). 
 The IUCN Red List is a global list of threatened and endangered species.
 
-IUCN Red List docs: http://apiv3.iucnredlist.org/api/v3/docs
+[IUCN Red List docs][docs]
 
-> important note: [redlistr][] is a different package - not working with the IUCN Red List API
+> What rredlist is not: [redlistr][] is a different package - not working with the IUCN Red List API; Furthermore, rredlist does not include support for the spatial API, described at
```

Comment:
this packages doesn't either though - https://github.com/ropensci/rredlist#no-spatial

Reply:
I'm actually removing this section hence my adding the sentence at the top. I thought it might be clearer to group the two things the rredlist package is not (rredlist is not redlistr and rredlist isn't a wrapper for the spatial stuff)

Reply:
ok